--- openair-cn/src/s6a/s6a_peer.c	2017-06-07 16:17:08.089045775 +0200
+++ opencells-mods/src/s6a/s6a_peer.c	2017-06-06 12:23:18.000000000 +0200
@@ -88,8 +88,6 @@
 s6a_fd_new_peer (
   void)
 {
-  char                                    host_name[100];
-  size_t                                  host_name_len = 0;
   int                                     ret = 0;
 #if FD_CONF_FILE_NO_CONNECT_PEERS_CONFIGURED
   struct peer_info                        info = {0};
@@ -100,18 +98,6 @@
     return RETURNerror;
   }
 
-  if (fd_g_config->cnf_diamid ) {
-    free (fd_g_config->cnf_diamid);
-    fd_g_config->cnf_diamid_len = 0;
-  }
-
-  DevAssert (gethostname (host_name, 100) == 0);
-  host_name_len = strlen (host_name);
-  host_name[host_name_len] = '.';
-  host_name[host_name_len + 1] = '\0';
-  strcat (host_name, (const char *)mme_config.realm->data);
-  fd_g_config->cnf_diamid = strdup (host_name);
-  fd_g_config->cnf_diamid_len = strlen (fd_g_config->cnf_diamid);
   OAILOG_DEBUG (LOG_S6A, "Diameter identity of MME: %s with length: %zd\n", fd_g_config->cnf_diamid, fd_g_config->cnf_diamid_len);
   bstring                                 hss_name = bstrcpy(mme_config.s6a_config.hss_host_name);
   bconchar(hss_name, '.');
