--- openair-cn/src/gtpv1-u/gtp_mod_kernel.h	2017-06-07 16:17:08.053047242 +0200
+++ opencells-mods/src/gtpv1-u/gtp_mod_kernel.h	2017-06-05 18:19:13.000000000 +0200
@@ -6,7 +6,7 @@
 int gtp_mod_kernel_tunnel_add(struct in_addr ue, struct in_addr gw, uint32_t i_tei, uint32_t o_tei);
 int gtp_mod_kernel_tunnel_del(uint32_t i_tei, uint32_t o_tei);
 
-int gtp_mod_kernel_init(int *fd0, int *fd1u, struct in_addr *ue_net, int mask, int gtp_dev_mtu);
+int gtp_mod_kernel_init(int *fd0, int *fd1u, struct in_addr *ue_net, int mask,spgw_config_t * spgw_config);
 void gtp_mod_kernel_stop(void);
 
 #endif /* FILE_GTP_MOD_KERNEL_SEEN */
