--- openair-cn/src/gtpv1-u/gtpv1u_task.c	2017-06-07 16:17:08.053047242 +0200
+++ opencells-mods/src/gtpv1-u/gtpv1u_task.c	2017-06-05 18:20:53.000000000 +0200
@@ -111,7 +111,7 @@
     gtp_mod_kernel_init(&sgw_app.gtpv1u_data.fd0, &sgw_app.gtpv1u_data.fd1u,
         &spgw_config->pgw_config.ue_pool_addr[i],
         spgw_config->pgw_config.ue_pool_mask[i],
-        spgw_config->pgw_config.ipv4.mtu_SGI);
+          spgw_config);
   }
   // END-GTP quick integration only for evaluation purpose
 
