--- openair-cn/src/common/common_defs.h	2017-06-07 16:17:08.037047894 +0200
+++ opencells-mods/src/common/common_defs.h	2017-06-06 21:09:43.000000000 +0200
@@ -109,11 +109,8 @@
 
 #define IPV4_STR_ADDR_TO_INT_NWBO(AdDr_StR,NwBo,MeSsAgE ) do {\
             struct in_addr inp;\
-            if ( inet_aton(AdDr_StR, &inp ) < 0 ) {\
-                AssertFatal (0, MeSsAgE);\
-            } else {\
-                NwBo = inp.s_addr;\
-            }\
+            AssertFatal (inet_aton(AdDr_StR, &inp ) , MeSsAgE);\
+            NwBo = inp.s_addr;\
         } while (0)
 
 #define NIPADDR(addr) \
